db/seeds/hhg_weights.json

```diff
@@ -182,10 +182,6 @@
         "weight": 70
       },
       {
-        "name": "Typewriter",
```

Comment:
Typewriter LOL

Reply:
You'd be surprised where typewriters turn up… 😐